app/controllers/api/v1/categories_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module V1
+    class CategoriesController < ApiController
+      def index
+        categories = Category.
```

Comment:
Is it really necessary to make it 3 lines here?

Reply:
No, I can change that.